roles/idm-host-cert/README.md

```diff
@@ -0,0 +1,100 @@
+Set of Roles
+============
+
+The ansible roles found in this directory are associated with creating and configuring idm host certs.
+
+First it logs into idm, creates a session, and then registers the host. Then, it generates a private key and csr. Next, it creates the cert, retrieves the old one, and overwrites it. As a final check, it prints out the certs.
+
+Requirements
+------------
+
+An IDM accout and ansible.
+
+Role Variables
+--------------
+| Variable | Description | Required | Defaults |
```

Comment:
Make sure to include the following two variables:
- `host_force_add` - defaults to `true`. 
- `api_version` - defaults to `2.213`. Add description that this is the API version as reported by the IdM (not the IdM version).